package.json

```diff
@@ -9,7 +9,7 @@
     "react-native",
     "webgl"
   ],
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
mmh I think the entry point still need to be lib/index.js https://unpkg.com/browse/react-native-webgl@0.8.0/lib/index.js , the babel-compiled version